csharp_dotnetcore/60.ConversationHistory/TranscriptStore.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Bot.Builder.Azure;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public class TranscriptStore
+    {
```

Comment:
code comments

Reply:
Not applicable anymore.

---
In reply to: [218236639](https://github.com/Microsoft/BotBuilder-Samples/pull/391#discussion_r218236639) [](ancestors = 218236639)